guides/common/modules/proc_adding-proxmox-details-to-a-compute-profile.adoc

```diff
@@ -12,34 +12,15 @@ Note that this procedure assumes you are using KVM/qemu-based virtualization.
 . Select a Proxmox compute resource.
 // FIXME: this procedure currently assumes you've selected KVM/qemu
 . In the *Type* list, select the kind of virtualization solution hosts will run on when created using this compute profile.
```

Comment:
```suggestion
. From the *Type* list, select a virtualization options:

* **LXC container**: description
* **KVM/Qemu server**: description
```
Not sure why "Qemu" is not all caps but that is outside the scope of your PR.

Reply:
I don't know either, but it is:

![image](https://github.com/user-attachments/assets/7117024c-1383-4ffa-94cf-d447c2c2fc4d)

Applied your suggestion with "option".